Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -716,17 +722,18 @@ - (BOOL)isCapturingStillImage {
 
 #pragma mark - Setters
 
-- (void)setCamera:(MTBCamera)camera {
```

Comment:
I agree with the changes you suggested. If we did decide to break backwards compatibility I would release a major version but I'd prefer not to!